include/ppx/bitmap.h

```diff
@@ -182,7 +184,7 @@ class Bitmap
 
 private:
     void   InternalCtor();
-    Result InternalInitialize(uint32_t width, uint32_t height, Bitmap::Format format, char* pExternalStorage);
+    Result InternalInitialize(uint32_t width, uint32_t height, Bitmap::Format format, uint32_t rowStride, char* pExternalStorage);
```

Comment:
Can you add a comment that if `rowStride` is zero, then the format's default row stride is used?

Reply:
Done. Added comments  for that block of functions to keep things uniform.